app/views/admins/show_facilities.html.erb

```diff
@@ -1,4 +1,4 @@
-Facilities
+Facilities | | <%= link_to "Back To Admin Page", admins_path() %>
```

Comment:
You don't need to add empty parentheses to your `index` path.

Reply:
removed ().